test/jdk/jdk/incubator/vector/FloatMaxVectorTests.java

```diff
@@ -609,7 +625,6 @@ static void assertDoubleBroadcastArraysEquals(float[] r, float[] a, float[] b, f
         }
     }
 
-
```

Comment:
This empty line should not get removed.

Reply:
Fixed.
Now it is in line with https://github.com/openjdk/jdk/blob/e6f23a90d4a53339a3c9c2b76fc5d317940e4472/test/jdk/jdk/incubator/vector/FloatMaxVectorTests.java#L720-L721